app/views/users/show/_about_frame.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="panel panel-default module">
+  <div class="panel-heading">
+    <b>About Me</b>
+    <% if can? :edit, user %>
+      <a class="rip-edit rip-edit-inline pull-right" onclick="$('#user-bio').click();">
+        <i class="fa fa-pencil"></i>
+      </a>
+    <% end %>
+  </div>
+  <% if can? :edit, user %>
+    <div class="panel-paragraphs">
+      <%= content_tag :p, user.bio, {
+        :id => 'user-bio',
+        :class => 'rest-in-place rip-bio',
+        :data => {
+          :formtype => 'textarea',
+          :url => user_path(user),
+          :object => 'user',
+          :attribute => 'bio',
+          :placeholder => 'Tell us a little about yourself.'
```

Comment:
Should we grey out the placeholder text in the about me module like in the assigned videos module?